dkg-gadget/src/gossip_engine/network.rs

```diff
@@ -390,23 +388,6 @@ impl<B: Block + 'static> GossipHandler<B> {
 	async fn handle_network_event(&self, event: Event) {
 		match event {
 			Event::Dht(_) => {},
-			Event::SyncConnected { remote } => {
-				let addr = iter::once(multiaddr::Protocol::P2p(remote.into()))
-					.collect::<multiaddr::Multiaddr>();
-				let result = self
-					.service
-					.add_peers_to_reserved_set(self.protocol_name.clone(), HashSet::from([addr]));
-				if let Err(err) = result {
-					self.logger.error(format!("Add reserved peer failed: {err}"));
-				}
-			},
-			Event::SyncDisconnected { remote } => {
-				self.service.remove_peers_from_reserved_set(
-					self.protocol_name.clone(),
-					iter::once(remote).collect(),
-				);
-			},
-
```

Comment:
Where are these events handlers gone? what are the alternatives?

Reply:
they moved it to separate SyncEvents, updated.